apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -80,6 +81,16 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
 
   return (
     <div className={moduleStyles.rightHeaderButtons}>
+      {projectPickerSettings && (
+        <Button
+          iconRight={{iconName: 'rotate'}}
+          size={'xs'}
+          color={'white'}
```

Comment:
@hannahbergam noting that if #65022 goes in before this, I'll need to update this to not have a color anymore, right?

Reply:
removed hard-coded color!

Reply:
Would it make sense to add an `ariaLabel` here, something like 'Change project (or output) type'?

Reply:
added label!